### ISSUE TITLE
"Super"node set up

### ISSUE THREAD

**Issue by milyway** [WhenLamboMoon/docker-zen-node#87]
Hi, 

Thanks to your help, I was able to get past the install.sh issue,
successfully generated the shield address, sent 1 ZEN to this address, and

docker exec zen-node gosu user zen-cli z_gettotalbalance
shows 

<img width="195" alt="dockerzennode2" src="https://user-images.githubusercontent.com/40526863/42981802-be0e30e4-8ba3-11e8-9ad4-0d498c64565d.png">

and 
docker exec zen-node gosu user zen-cli getinfo
shows

<img width="265" alt="dockerzennode" src="https://user-images.githubusercontent.com/40526863/42981639-bb8ee4c2-8ba2-11e8-998d-5c64a3cf92f5.png">

but when I type docker logs zen-secnodetracker, I get error messages saying

ERROR zend get t-address
connect ENETUNREACH 127.0.0.0:18231 - Local (0,0,0,0:0)
ERROR zend get t-address
connect ENETUNREACH 127.0.0.0:18231 - Local (0,0,0,0:0)
ERROR zend get t-address
connect ENETUNREACH 127.0.0.0:18231 - Local (0,0,0,0:0)
ERROR zend get t-address
connect ENETUNREACH 127.0.0.0:18231 - Local (0,0,0,0:0)

I'd greatly appreciate any help, do you think supernode needs a different set up from secure node? 
I have 500 ZEN in the wallet address I used for <stakeaddr>, and chose super for <nodetype>

Thank you very much.

**Comment by 333miiko** [comment]
Can you edit /mnt/zen/config/zen.conf and change the second line  to:

rpcallowip=127.0.0.1

Then restart:

```
docker restart zen-node
docker restart zen-secnodetracker
```

**Comment by milyway** [comment]
I followed your instructions, and it says

<img width="771" alt="dockerzennode3" src="https://user-images.githubusercontent.com/40526863/42983226-2c22d86c-8bab-11e8-90c8-d1398c7edfbe.png">

so after multiple error messages, it seems to start, but the node type is "secure" not "super"

also zencash site says certificate is not valid

https://securenodes2.na.zensystem.io/nodes/84272/detail

**Comment by Sha01in** [comment]
Edit /mnt/zen/secnode/config.json so that first 4 lines are the following then restart node and tracker.

{
 "active": "super",
 "super": {
  "nodetype": "super",

**Comment by slayorktc** [comment]
Change your config.json to xns1 from ts1

**Comment by slayorktc** [comment]
Let me know if challeges work correctly for you

**Comment by slayorktc** [comment]
Finally got everything on the super working. Challenges and all. Let me know if you still need help.

**Comment by milyway** [comment]
I've tried to change it to xns1 but did not work.
Would you mind sharing me the config.json?

This is what I did, but did not work

<img width="960" alt="dockerzennode5" src="https://user-images.githubusercontent.com/40526863/43030960-0f87b016-8c5e-11e8-8951-5ab5ce881008.png">

**Comment by milyway** [comment]
Thank you for the information.

So this is what I did instead for the config.json,

<img width="960" alt="dockerzennode6" src="https://user-images.githubusercontent.com/40526863/43031438-7cde3d66-8c67-11e8-8803-736a5c85f97f.png">

and this is what I got

<img width="942" alt="dockerzennode7" src="https://user-images.githubusercontent.com/40526863/43031458-eb5e8ce6-8c67-11e8-8e28-deb01d175ef7.png">


It's saying cannot find a registration and cannot authenticate....

**Comment by slayorktc** [comment]
Yeah I had that as well.. Not sure if it was related to nodeid or not. Drop that in the discord and people can help sort that out.

**Comment by milyway** [comment]
If you had the same issue, how did you get yours finally work ?

**Comment by KenJusticeJr** [comment]
I think I had the same issue, and was able to get it to work. Check out Issue #102 to see how I fixed it. (Sorry for creating a new issue instead of working on this one...I didn't realize that they were related.)
